biz.aQute.junit.platform.reporting/bnd.bnd

```diff
@@ -0,0 +1,4 @@
+-includeresource.reporting: @${repo;org.junit.platform:junit-platform-reporting;${junit-platform-reporting}}
```

Comment:
New project bnd.bnd files need to include jdt.bnd.

https://github.com/bndtools/bnd/blob/79a9e4c4ff78c030a58efc381b6ce05f5cf2cb8e/biz.aQute.bndlib/bnd.bnd#L1-L2

Reply:
I didn't think it mattered because `biz.aQute.junit.platform.reporting` doesn't contain any source, but I have included now to be sure.